tls/s2n_resume.c

```diff
@@ -65,6 +66,27 @@ static int s2n_tls12_serialize_resumption_state(struct s2n_connection *conn, str
     return 0;
 }
 
+static S2N_RESULT s2n_tls13_serialize_keying_material_expiration(struct s2n_connection *conn,
+        uint64_t now, struct s2n_stuffer *out)
+{
+    RESULT_ENSURE_REF(conn);
+    RESULT_ENSURE_REF(out);
+
+    if (conn->mode != S2N_SERVER) {
+        return S2N_RESULT_OK;
+    }
+
+    struct s2n_psk *chosen_psk = conn->psk_params.chosen_psk;
```

Comment:
Please move this down to have it closer to the null-check on line 82 and not be separated.